core/src/main/java/binnie/core/machines/storage/Compartment.java

```diff
@@ -9,6 +9,7 @@
 import binnie.core.machines.IMachineType;
 import binnie.core.machines.MachinePackage;
 import binnie.core.machines.TileEntityMachine;
+import static ic2.core.network.DataEncoder.EncodedType.ItemStack;
```

Comment:
?

Reply:
I must have tried to auto import a class to fix things but the minecraft directory wasn't found so it went for a class with the same name.

Reply:
fixed